documentation/how-to/outside-layers.en.md

```diff
@@ -5,21 +5,161 @@ tx_slug: documentation_how-to_outside-layers
 
 # Shared local datasets
 
-It is possible to use a layer which is stored outside the project folder.
+It is possible to use a layer which is stored outside the project folder. This is useful if a basemap (e.g., a large orthophoto) or large, unmodified vector data is used in more than one project. By sharing datasets, features and their attributes remain identifiable across projects.
 
-This is useful if a basemap is used in more than one project, for example a large orthophoto.
-In this case, you can share this dataset among different projects.
-Also large vector data files that are not modified can be shared that way. The features and their attributes remain identifiable.
+There are two main approaches to using shared local datasets: a manual copying method for direct use in QField, and a more integrated approach using QFieldCloud which leverages QGIS's Localized Data Paths.
+
+## Manual Copying of Shared Datasets for QField
+
+This method is suitable for users who want to manually manage shared files on their devices, often for basemaps, without using QFieldCloud for these specific files.
 
-## Add a layer outside of exported project folder
 :material-monitor: Desktop preparation
 
-In QGIS, open  *Options > Data Sources > Localized Data Paths*.
-In there, add the path to the external data.
+In QGIS, it's good practice set where your shared datasets are located.
+1. Open QGIS.
+2. Go to *Options > Data Sources > Localized Data Paths*.
+3. Add the path(s) to your external data.
+This primarily helps QGIS itself locate these layers (e.g., if you move projects between computers or use relative paths) and is **essential for the QFieldCloud workflow described later.**
+
+!![QGIS Options Dialog showing Data Sources and Localized Data Paths setup](../assets/images/external_path.png)
+
+### Using Manually Copied Datasets in QField
+
+To use these shared datasets manually in QField (e.g., as basemaps without using QFieldCloud for them):
```

Comment:
Manually copying files can also be used by cloud projects, we shouldn't imply otherwise.

```suggestion
To use these shared datasets manually in QField, locate the QField app directories on your device:
```